src/client/core-client.ts

```diff
@@ -92,6 +99,17 @@ interface GetServiceReply {
     service: Service
 }
 
+interface ServiceLogsRequest {
+    serviceID: string
+    dependencies: string[]
+}
+
+interface LogData {
+    dependency: string
+    type: any
+    data: any
```

Comment:
i think this can be updated as uint8 array but i'm not sure if bytes are encoded in 8bit by protobuf

Reply:
another PR/issue